test/dependencies.js

```diff
@@ -47,22 +47,12 @@ describe("dependencies", function() {
 				"/a/b/package.json",
 				"/a/package.json",
 				"/package.json",
-				"/a/node_modules/package.json",
```

Comment:
✅ These package.json origin from trying to resolve as single file module. We don't need them